### PR TITLE
Fix errors in test command on 3D and multichannel

### PR DESCRIPTION
**Pull request by andreanne-lemay**
When running the test command, some errors occurred on 3D models or multichannel.

### CODE REVIEW

File: ivadomed/main.py [hunk 1]
```diff
@@ -799,36 +803,37 @@ def cmd_test(context):
...
-
-        fname_unc_vox = os.path.join(path_3Dpred, subj_acq+'_unc-vox.nii.gz')
-        fname_unc_struct = os.path.join(path_3Dpred, subj_acq+'_unc.nii.gz')
-        if not os.path.isfile(fname_unc_vox) or not os.path.isfile(fname_unc_struct):
-           # compute voxel-wise uncertainty map
-           voxelWise_uncertainty(fname_pred_lst, fname_unc_vox)
-
-           # compute structure-wise uncertainty
-           structureWise_uncertainty(fname_pred_lst, fname_pred, fname_unc_vox, fname_unc_struct)
+    if context['uncertainty']['epistemic'] or context['uncertainty']['aleatoric']:
```
Comment (andreanne-lemay) [inline ivadomed/main.py]: Without this condition, fname_pred_list is empty which triggers an error in combine_predictions.

Comment (charleygros) [reply ivadomed/main.py]: Indeed :-) Thanks!

File: ivadomed/main.py [hunk 2]
```diff
@@ -741,7 +741,7 @@ def cmd_test(context):
             rdict['gt'] = preds.cpu()
             batch.update(rdict)
 
-            if batch["input"].shape[1] > 1:
+            if batch["input"].shape[1] > 1 and not i_monteCarlo:
```
Comment (charleygros) [inline ivadomed/main.py]: Not sure to understand here: why the first iteration of MC is not taking only second channel?

Comment (andreanne-lemay) [reply ivadomed/main.py]: Because it is the same batch across the for loop and the operation needs to be done only once :)

Comment (charleygros) [reply ivadomed/main.py]: oooo! makes sense!

**Pull request merged**